packages/cli/openapi-ir-to-fern/src/__test__/__snapshots__/superagent.test.ts.snap

```diff
@@ -487,7 +487,14 @@ exports[`open api parser superagent docs 1`] = `
               "body": {
                 "properties": {
                   "authorization": "optional<map<string, unknown>>",
-                  "from_page": "optional<integer>",
+                  "from_page": {
+                    "default": [
+                      1,
+                    ],
+                    "docs": undefined,
+                    "type": "optional<integer>",
+                    "validation": undefined,
+                  },
```

Comment:
would be great if these `undefined` keys didnt show up in the generated fern definition to keep it as compact as possible

Reply:
Nice catch, it looks like we've had this issue for a while, but I'll fix it here ([link](https://github.com/fern-api/fern/blob/9ae2f2c75b659551205812148eaa1c38ec2e8165/packages/cli/openapi-ir-to-fern/src/buildTypeReference.ts#L458)).